specification/ml/_types/Model.ts

```diff
@@ -49,6 +49,9 @@ export class ModelSnapshotUpgrade {
   job_id: Id
   snapshot_id: Id
   state: SnapshotUpgradeState
+  /**
+   * @availability stack
+   */
   node: DiscoveryNode
```

Comment:
I believe all types which have had this `@availability stack` comment added to should also be make optional to communicate that this item will sometimes not exist. e.g.
```suggestion
  node?: DiscoveryNode
```

Reply:
For `DiscoveryNode` it's just this one place. I wouldn't like to do this for one of the more common APIs like open job or start datafeed, as it could break client code. However, since upgrading model snapshots is so esoteric I doubt anyone will be affected so I'll make the change.